tests/admin_app_tests/endpoint_tests/posts_test.go

```diff
@@ -60,10 +65,250 @@ func TestIndexPing(t *testing.T) {
 	assert.Equal(t, response.Id, 0)
 }
 
-// TODO : Test request without excerpt
+func TestPostPostSuccess(t *testing.T) {
+	databaseMock := mocks.DatabaseMock{
+		AddPostHandler: func(string, string, string) (int, error) {
+			return 0, nil
+		},
+	}
+
+	postData := admin_app.AddPostRequest{
+		Title:   "Title",
+		Excerpt: "Excerpt",
+		Content: "Content",
+	}
+
+	router := admin_app.SetupRoutes(app_settings, databaseMock)
+	responseRecorder := httptest.NewRecorder()
+
+	body, _ := json.Marshal(postData)
+	request, _ := http.NewRequest(http.MethodPost, "/posts", bytes.NewBuffer(body))
+
+	router.ServeHTTP(responseRecorder, request)
+
+	assert.Equal(t, 200, responseRecorder.Code)
```

Comment:
Just minor change: let's use the `http.StatusOK` instead of 200, just for clarity on what the response means